### ISSUE TITLE
Allow const static str's to be used as operation_id

### ISSUE THREAD

**Issue by djrenren** [juhaku/utoipa#471]
**TL;DR** It would be useful if the `operation_id` option on the `#[path]` macro could be a const variable instead of just a static string like so:

```rust
const MY_ID: &str = "some_id";

#[utoipa::path(
    ...
    operation_id = MY_ID
)]
fn foo(...) { ... }
```

**Why**
I'm currently writing an API that runs on an AWS Lambda behind API Gateway. There's this really nice feature of API Gateways where you give it an openapi spec and when it invokes your lambda, it tells you the operation id that corresponds to this request. So you can easily write a router like so:

```rust

match request_context.operation_name.as_deref() {
   route::MY_ID => foo(...)
}
```

Currently, I have to keep the string id in sync between my const definition and the `#[path]` macro invocation. It would be nice to just have a single source of truth.

Is this something you'd be open to a PR for?

**Comment by juhaku** [comment]
Sure any contributions are welcome :slightly_smiling_face:
